packages/gestalt/src/DropdownItem.jsdom.test.js

```diff
@@ -0,0 +1,60 @@
+// @flow strict
+import React from 'react';
+import { create } from 'react-test-renderer';
```

Comment:
Is there a reason to use both testing libraries here?

Reply:
Nope, just updated to only use one